projects/xUnit/objects/objTestAsyncDraw/Create_0.gml

```diff
@@ -0,0 +1,97 @@
+/// @description Begin running test, and declare some variables and functions for draw buffer comparisons
+// The test name to use as a prefix for any saved/loaded files during the test.
+testFileName = ""
```

Comment:
we can probably refactor this out from the test handler and put it into some reusable space maybe... "utils" ?